test/java/seedu/addressbook/data/AddressBookTest.java

```diff
@@ -79,8 +79,8 @@ public void setUp() throws Exception {
     @Test
     public void addPerson_emptyAddressBook() throws Exception {
         emptyAddressBook.addPerson(bobChaplin);
-        Person bobChaplin_confirmPerson = emptyAddressBook.getAllPersons().iterator().next();
-        assertTrue(bobChaplin.equals(bobChaplin_confirmPerson));
+        Person bobChaplinConfirmPerson = emptyAddressBook.getAllPersons().iterator().next();
+        assertTrue(bobChaplin.equals(bobChaplinConfirmPerson));
```

Comment:
This naming style (using `_` as prefix/suffix for local variables), although allowed by our coding standard, is not consistent with the rest of the code base.